pkg/utils/image/infos.go

```diff
@@ -6,10 +6,10 @@ import (
 
 	"github.com/distribution/distribution/reference"
 	"github.com/kyverno/kyverno/pkg/config"
-	"sigs.k8s.io/controller-runtime/pkg/log"
+	"github.com/kyverno/kyverno/pkg/logging"
 )
 
-var logger = log.Log.WithName("image")
+var logger = logging.GlobalLogger().WithName("image")
```

Comment:
The issue is about fixing logs in Kyverno CLI. I amn't sure if this needs to be changed too but I can't see logs fixed in CLI. The code exists at `cmd/cli/kubectl-kyverno`.

Reply:
@MariamFahmy98 Do I have to fix logs for each command of kyverno cli files?

Reply:
@siddhikhapare yes those would be the required changes.
Ex: https://github.com/kyverno/kyverno/blob/main/cmd/cli/kubectl-kyverno/apply/apply_command.go#L32
Here we are using `sigs.k8s.io/controller-runtime/pkg/log`. We will want to change that as the issue suggests.